src/cascadia/TerminalSettingsEditor/ColorSchemes.cpp

```diff
@@ -43,6 +43,16 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
             _layoutUpdatedRevoker.revoke();
 
             ColorSchemeListView().Focus(FocusState::Programmatic);
+
+            for (const auto scheme : _ViewModel.AllColorSchemes())
+            {
+                if (scheme.IsDefaultScheme())
+                {
+                    winrt::hstring newName{ fmt::format(L"{} ({})", scheme.Name(), RS_(L"ColorScheme_DefaultTag/Text")) };
```

Comment:
```suggestion
                    const auto newName = fmt::format(L"{} ({})", scheme.Name(), RS_(L"ColorScheme_DefaultTag/Text"));
```

You can also use `{}` instead of `=` if you prefer. This file seems to use a mix of both styles.

WinRT generated functions accept a `winrt::param::hstring` instead of a `winrt::hstring`, the former of which are "borrowed" strings. It allows you to pass a non-hstring string without making any additional copies and is very convenient to use, because any null-terminated string implicitly (!) converts to it. = No casting needed.